test/integration/dataobjectsTest.js

```diff
@@ -0,0 +1,317 @@
+import should from 'should/as-function';
+import Promise from 'bluebird';
+import _ from 'lodash';
+import Syncano from '../../src/syncano';
+import {ValidationError} from '../../src/errors';
+import {suffix, credentials} from './utils';
+
+describe('Dataobject', function() {
+  this.timeout(15000);
+
+  let connection = null;
+  let Class = null;
+  let Instance = null;
+  let DataObject = null;
+  const instanceName = suffix.get('instance');
+  const className = suffix.get('class');
+  const data = {
+    name: className,
+    instanceName: instanceName,
+    description: suffix.get('description'),
+    schema: [
+      { name: "title", type: "string" },
+      { name: "author", type: "string" }
+    ]
+  };
+
+  before(function() {
+    connection = Syncano(credentials.getCredentials());
+    Instance = connection.Instance;
+    Class = connection.Class;
+    DataObject = connection.DataObject;
+
+    return Instance.please().create({name: instanceName});
+  });
+
+  after(function() {
+    return Instance.please().delete({name: instanceName});
+  });
+
+  beforeEach(function() {
+    return Class.please().create(data);
```

Comment:
do we need to create new class before each tests? one with all types of fields in schema should do the trick :)